README.md

```diff
@@ -142,6 +142,12 @@ And to disable it:
 L.PM.setOptIn(false);
 ```
 
+With opt-in you need to specify `pmIgnore: false` in map options on creation:
```

Comment:
Can you please change it to something like "If you have enabled opt-in before you init the map, you need to specify `pmIgnore: false` in the map options."